### PR TITLE
Update Russian and Ukrainian translations

### PR DESCRIPTION
**Pull request by Vovkiv**
Finished now, but didn't tested that much except quickly glancing

### CODE REVIEW

File: translations/ru.po [hunk 1]
```diff
@@ -84,63 +84,62 @@ msgid "Insert After"
 msgstr "Вставить после"
 
 #: src/config_classes/Formatter.gd
-#, fuzzy
 msgid "Always"
-msgstr "Всегда активно."
+msgstr "Всегда активно"
```
Comment (MewPurPur) [inline translations/ru.po:88]: It shouldn't be "always active". That string is retired, this is in the context of "Always / Sometimes / Never" sorts of dropdowns.

File: translations/ru.po [hunk 2]
```diff
@@ -84,63 +84,62 @@ msgid "Insert After"
...
 
 #: src/data_classes/ElementG.gd
 msgid "This group has only one element."
-msgstr ""
+msgstr "В этой группе только один элемент."
 
 #: src/data_classes/ElementLinearGradient.gd
 msgid "No id attribute defined."
-msgstr ""
+msgstr "Атрибут идентефикатора не определен."
```
Comment (MewPurPur) [inline translations/ru.po:128]: "id" is the name of the attribute.

Comment (MewPurPur) [reply translations/ru.po:128]: Oh I see, yeah this string has a version with quotation marks too. That one is valid, and I'll erase this one.

File: translations/ru.po (hunk 1 above)
Comment (Vovkiv) [reply translations/ru.po:88]: I guess I just missed string haha

**Comment by MewPurPur** [comment]
Any update? I partially finished translation is also fine, it's still better than none.

**Comment by MewPurPur** [comment]
Thanks!

**Pull request merged**